internal/app/machined/pkg/runtime/v1alpha1/v1alpha1_sequencer.go

```diff
@@ -287,6 +287,20 @@ func (*Sequencer) Boot(r runtime.Runtime) []runtime.Phase {
 	return phases
 }
 
+func parseWipeParamString(wipe *string) []runtime.Phase {
+	if wipe == nil || *wipe == "" {
+		return nil
+	}
+
+	phases := PhaseList{}
+
+	if *wipe == "system" {
+		return phases.Append("wipeSystemDisk", ResetSystemDisk).Append("reboot", Reboot)
+	}
+
+	return phases.Append("wipeSystemDisk", ResetSystemDiskPartitions(*wipe)).Append("reboot", Reboot)
```

Comment:
same way I had a comment here that we can skip `Reboot` if we move it up to the `Initialize` sequence if `BOOT` is not wiped

Reply:
Done.